### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tungntEmotiv**
- Update and fix bugs of EmotivTools, Edfheaderconverter for both Mac and Windows
- Add and fix bugs some examples in python, java 
- Update android for sdk (still wait @nguoithichkhampha  upload library to server)
- Fix some merge issue

### CODE REVIEW

File: examples_basic/Java/src/com/emotiv/examples/EmoStateLogger/EmoStateLogger.java [hunk 1]
```diff
@@ -72,12 +72,6 @@ public static void main(String[] args) {
 							.IS_FacialExpressionIsRightWink(eState) == 1)
 						System.out.println("RightWink");
                     
-					System.out.print("MentalCommandGetCurrentAction: ");
```
Comment (hoangphamemotiv) [inline examples_basic/Java/src/com/emotiv/examples/EmoStateLogger/EmoStateLogger.java]: did we remove it? it is a part of emostate

File: examples_extra_prime/Android/EEGLogger/.idea/gradle.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hoangphamemotiv) [inline examples_extra_prime/Android/EEGLogger/.idea/gradle.xml]: we need to remove all changes of examples, librarys relate to Android

Comment (hoangphamemotiv) [reply examples_extra_prime/Android/EEGLogger/.idea/gradle.xml]: @tungntEmotiv @duytan90

File: examples_basic/Java/src/com/emotiv/examples/EmoStateLogger/EmoStateLogger.java (hunk 1 above)
Comment (tungntEmotiv) [reply examples_basic/Java/src/com/emotiv/examples/EmoStateLogger/EmoStateLogger.java]: okie, i think we should revert to old implement for emostate

**Pull request merged**